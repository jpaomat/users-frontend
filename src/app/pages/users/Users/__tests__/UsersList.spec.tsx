import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
const { Users } = require('../Users');

describe('<UserList /> component', () => {
    it('should render component', () => {
		const {container} = render(
			<MemoryRouter>
				<Users />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
