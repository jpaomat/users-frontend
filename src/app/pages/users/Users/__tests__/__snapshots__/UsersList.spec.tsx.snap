// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserList /> component should render component 1`] = `
<div>
  <div>
    Users
  </div>
</div>
`;
